package com.contatos.gerenciadordecontatos.errors;

public class ContatoNotFoundException extends RuntimeException{

    public ContatoNotFoundException(Long idContato){
        super("Contato n√£o foi encontrado com ID: " + idContato);
    }
}
